packages/components/src/DataList/components/DataListHeaderTile/DataListHeaderTile.tsx

```diff
@@ -60,6 +60,7 @@ export function DataListHeaderTile<T extends DataListObject>({
       {isSortable && (
         <DataListSortingArrows
           order={sortingState?.key === headerKey ? sortingState.order : "none"}
+          columnKey={headerKey}
```

Comment:
What are your thoughts on changing `columnKey` to `headerKey` to match the prop in `DataListHeaderTile`?